buildkite/test_plans.go

```diff
@@ -0,0 +1,49 @@
+package buildkite
+
+import "fmt"
+
+type TestPlansService struct {
+	client *Client
+}
+
+type Case struct {
+	Path              *string `json:"path,omitempty" yaml:"path,omitempty"`
```

Comment:
I don't see a reason this needs to be a pointer. The path will always be in the response.